cbt_export_changes.py

```diff
@@ -102,23 +121,24 @@ def main():
                                 "CBT example")
 
     try:
-        collect_host_certificates(session)
         vdi_ref = session.xenapi.VDI.get_by_uuid(args.vdi_uuid)
         last_snapshot_ref = session.xenapi.VDI.get_by_uuid(args.snapshot_uuid)
         new_snapshot_ref = session.xenapi.VDI.snapshot(vdi_ref)
         bitmap = session.xenapi.VDI.list_changed_blocks(last_snapshot_ref,
                                                           new_snapshot_ref)
-        download_changed_blocks(bitmap,
-                        session.xenapi.VDI.get_nbd_info(new_snapshot_ref)[0],
+        # get_nbd_info may return the details for multiple networks, for this

```

Comment:
A minor point, but instead of "networks" it would be better to say "addresses" (since even if only one of the pool's networks has the nbd purpose, this network might have multiple PIFs on each of multiple hosts).